### DISCUSSION TITLE
تغییر آی پی سرور در هایدیفای

### DISCUSSION THREAD

**Discussion by masoudimo** [hiddify/Hiddify-Manager#615] [Q&A سوالات رایج]
درود در پنل های قدیمی x-ui ما به راحتی می توانستیم با تغییر آی پی در هتزنر و سپس از کلادفلر آی پی فیلتر شده سرور را عوض کنیم. ولی ظاهرا در هایدیفای اگر بخواهیم آی پی اصلی سرور را عوض کنیم باید کل سرور مجدد نصب بشه. تصور من درسته؟

**Comment by dimbalato** [comment]
سلام. وقت بخیر
تا جایی که من میدونم، بستگی به سرویس دهنده سرور شما داره. بعضی از سرویس دهنده ها امکان تغییر ای پی سرور رو میدن مثل هتزنر یا پارس پک. 
اما اگر این امکان رو نداشتین میتونین از تنظیمات پنل، فایل پشتیبان رو دانلود کنید و بعداز نصب مجدد پنل روی سرور جدید همین تنظیمات رو بارگذاری کنید.
